torch/csrc/Module.cpp

```diff
@@ -888,6 +888,25 @@ PyObject* THPModule_setDeterministicAlgorithms(
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPModule_setAllowTF32Mkldnn(PyObject* _unsued, PyObject* arg) {
+  HANDLE_TH_ERRORS
+  TORCH_CHECK(
+      PyBool_Check(arg),
+      "set_allow_tf32_cublas expects a bool, "
```

Comment:
`set_allow_tf32_cublas` -> `set_allow_tf32_mkldnn`.

Reply:
Thanks for the reminding, have modified